src/View/ViewableData.php

```diff
@@ -586,6 +589,35 @@ public function Me()
         return $this;
     }
 
+    /**
+     * Return the directory if the current active theme (relative to the site root).
+     *
+     * This method is useful for things such as accessing theme images from your template without hardcoding the theme
+     * page - e.g. <img src="$ThemeDir/images/something.gif">.
+     *
+     * This method should only be used when a theme is currently active. However, it will fall over to the current
+     * project directory.
+     *
+     * @return string URL to the current theme
+     * @deprecated 4.0.0..5.0.0 Use $resourcePath or $resourceURL template helpers instead
+     */
+    public function ThemeDir()
+    {
+        Deprecation::notice('5.0', 'Use $resourcePath or $resourceURL template helpers instead');
+        $themes = SSViewer::get_themes();
+        foreach ($themes as $theme) {
+            // Skip theme sets
+            if (strpos($theme, '$') !== false) {
```

Comment:
this would mean you could have a `$` in the middle of your path and it would skip, could this have the check at the beginning of the name `strpos() === 0` instead?

I guess this is edge-case so not a dealbreaker for me

Reply:
If you have a `$` in the middle of your theme then it'd probably be skipped because it's a bad theme.